### PR TITLE
kbounce: init 20.04.3

### PR DESCRIPTION
**Pull request by freezeboy**
###### Motivation for this change

new package

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by freezeboy** [comment]
@ttuegel Do you know how to deal with the 30 PRs I proposed ? they are all basically using the same principle

**Comment by Lassulus** [comment]
can you squash the commits?

**Comment by freezeboy** [comment]
Collecting the different kdeApps in this PR and reorganizing the inherit in top-level/all-applications to avoid future conflicts

File: pkgs/applications/kde/kalzium.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    maintainers = with maintainers; [ freezeboy ];
+    license = licenses.gpl2Plus;
+    platforms = platforms.linux;
+  };
+  nativeBuildInputs = [
+    extra-cmake-modules
+  ];
+  buildInputs = [
+    qtscript
+    #avogadro
```
Comment (ttuegel) [inline pkgs/applications/kde/kalzium.nix:17]: Is this dependency needed?

**Review by ttuegel — APPROVED** [review]
This looks good to me, but I wondered about the dependency of `kalzium` on `avogadro`.

File: pkgs/applications/kde/kalzium.nix (hunk 1 above)
Comment (freezeboy) [reply pkgs/applications/kde/kalzium.nix:17]: Currently I didn't succeed compiling kalzium with avogadro support (dont really know what it does in fact, just saw the reference in the CMakeLists.txt)

Do you want me to remove the reference ?

**Comment by peterhoeg** [comment]
#95893 was merged to staging, so I changed the base here as well.

File: pkgs/applications/kde/kalzium.nix (hunk 1 above)
Comment (ttuegel) [reply pkgs/applications/kde/kalzium.nix:17]: No, that's OK.

File: pkgs/applications/kde/kigo.nix [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{ mkDerivation, lib, extra-cmake-modules, kdoctools, ki18n, kio, libkdegames, knewstuff }:
+
+mkDerivation {
+  name = "kigo";
+  meta = with lib; {
+    homepage = "https://kde.org/applications/en/games/org/kde.kigo";
+    description = "Kigo est une implémentation libre du jeu de Go";
```
Comment (peterhoeg) [inline pkgs/applications/kde/kigo.nix:7]: @freezeboy - any reason this is in French?

Comment (freezeboy) [reply pkgs/applications/kde/kigo.nix:7]: wrong copy paste I guess, I should start a new PR from staging ?

**Comment by peterhoeg** [comment]
>   wrong copy paste I guess, I should start a new PR from staging ?

Either that or just wait until it hits master and then do the PR - it isn't critical.

**Pull request merged**